### PR TITLE
fix pyportal and add metro m4 wifi

### PR DESCRIPTION
**Pull request by ladyada**

### CODE REVIEW

**Comment by ladyada** [comment]
no rush :)

File: boards/metro_m4_wifi/board_config.h [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#ifndef BOARD_CONFIG_H
+#define BOARD_CONFIG_H
+
+#define VENDOR_NAME "Adafruit Industries"
+#define PRODUCT_NAME "Metro M4 Express WiFi"
+#define VOLUME_LABEL "METROM4BOOT"
+#define INDEX_URL "http://adafru.it/"
+#define BOARD_ID "SAMD51J19A-Metro-v0"
```
Comment (tannewt) [inline boards/metro_m4_wifi/board_config.h]: Want to update this? I imagine it's the same as the original metro.

Comment (tannewt) [reply boards/metro_m4_wifi/board_config.h]:
```suggestion
#define BOARD_ID "SAMD51J19A-Metro-WiFi-v0"
```

File: boards/metro_m4_wifi/board_config.h [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#ifndef BOARD_CONFIG_H
+#define BOARD_CONFIG_H
+
+#define VENDOR_NAME "Adafruit Industries"
+#define PRODUCT_NAME "Metro M4 Express WiFi"
```
Comment (tannewt) [inline boards/metro_m4_wifi/board_config.h]:
```suggestion
#define PRODUCT_NAME "Metro M4 Airlift"
```

**Comment by ladyada** [comment]
airlifted!

**Pull request merged**